### PR TITLE
back to 5 lines

### PR DESCRIPTION
**Pull request by nadimalabbas**
add 4th and 5th line

### CODE REVIEW

File: new.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 A new line for PR
 A 2nd line
 a 3rd line
+4th line
```
Comment (nadtesttest) [inline new.txt:4]: put a comma

**Review by nadtesttest — CHANGES_REQUESTED** [review]
I requested a comma

File: new.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+A new line for PR
+A 2nd line
+a 3rd line
+4th line
+5th line
```
Comment (nadtesttest) [inline new.txt:5]: needs a full stop

**Review by nadtesttest — CHANGES_REQUESTED** [review]
requested full stop

File: new.txt (hunk 2 above)
Comment (nadimalabbas) [reply new.txt:5]: ok will do

**Pull request closed without merging**